docs/operation/troubleshooting.md

```diff
@@ -0,0 +1,79 @@
+---
+id: troubleshooting
+title: Troubleshooting
+sidebar_label: Troubleshooting
+---
+
+This section contains information to help you identify and resolve errors you may encounter.
+
+### Known errors and issues
+
+#### lookup failed for scheduledEventID to activityID
```

Comment:
This and other pieces in here are meant to be monospaced extracts from log statements. They do not serve as good documentation headings. They are basically like code.